.gitignore

```diff
@@ -41,3 +41,4 @@ _infrastructure/tests/build
 node_modules
 
 .sublimets
+.settings/launch.json

```

Comment:
This if file created by Visual Studio Code(the new editor), I've opened DefinitelyTyped folder with it, so it created one(.settings/launch.json). 
Shell I remove it or something?